front/temporal/labs/activities.ts

```diff
@@ -100,52 +63,29 @@ export async function retrieveNewTranscriptsActivity(
     return [];
   }
 
-  // We only support google_drive for now.
-  if (transcriptsConfiguration.provider !== "google_drive") {
-    localLogger.error(
-      {},
-      "[retrieveNewTranscripts] Provider not supported. Stopping."
-    );
-
-    return [];
-  }
+  const transcriptsIdsToProcess: string[] = [];
 
```

Comment:
lets use a switch case with a default: assertNever() to make sure to update this piece of code whenever we add a new transcript provider.